test-js.sh

```diff
@@ -1,25 +1,6 @@
 #!/bin/bash
 
-while getopts 'c:p:' flag; do
-	case "${flag}" in
-		c) cwdiswppslinter=${OPTARG} ;;
-		p) plugindirname=${OPTARG} ;;
-	esac
-done
+# Run setup.
+. ./setup.sh
 
-# Get the absolute path to the plugin we want to check.
-if [ "$cwdiswppslinter" = "1" ]; then
-	plugindir="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath "$0")" )" )" )" )/$plugindirname"
-	wpcontentdir="./../../../../"
-	scriptsdir="$(dirname "$(realpath "$0")" )/"
-else
-	cwdiswppslinter=0
-	plugindir="$(dirname "$(dirname "$(realpath "$0")" )" )"
-	wpcontentdir="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath $0)" )" )" )" )"
-	scriptsdir="$plugindir/wpps-scripts/"
-fi
-
-cd "$wpcontentdir"
-sh "${scriptsdir}install-script-dependencies.sh" -c $cwdiswppslinter
-
-npm run test:js $plugindir
+npm run test:js -- --roots $plugindir
```

Comment:
This is the main thing that fixed `npm run test:js`

Reply:
Interesting! What does this do?

Reply:
Before, it was evaluating `$plugindir` as a pattern, like `test/**/*.js`, not as a file to run tests in.

It looks like `--roots` accepts the files that it should run tests in.